beads_web/src/main/java/com/beads/web/vaadin/view/order_item/component/OrderItemTableModel.java

```diff
@@ -0,0 +1,57 @@
+package com.beads.web.vaadin.view.order_item.component;
+
+import static java.util.Objects.requireNonNull;
+import com.beads.model.domain.Order;
+import com.beads.model.domain.OrderItem;
+import com.vaadin.data.util.BeanItemContainer;
+import java.util.List;
+
+public class OrderItemTableModel extends BeanItemContainer<OrderItem> {
```

Comment:
Do we need this ?

Reply:
if you mean `order` we use it in populateContainerFullList()